README.md

```diff
@@ -319,27 +327,38 @@ If you found a bug or want to suggest a new feature, please file a Github issue.
 
 ## Changelog
 
+- 2022-08-03 `v3.0.0`
+    - this version is not fully backward compatible because of differences in migration commands
+    - added sync of customers who made an order without account
+    - enhanced CLI: removing three `migrate-...` commands and replace them with one `migrate` with several options
+    - added migration of metadata: as metadata from commercetools side are considered `custom fields` in case of `orders` and `customers` and `attributes` in case of `products`
+    - fixed problem with deleting other api extensions while api extension registration
+    - readme update
+
 - 2022-08-02 `v2.0.0`
     - version v2.x is not fully backward compatible with version v1.x, please refer to [Migration from v1.x.x to v2.x.x](#migration-from-v1xx-to-v2xx) section
     - fixing the issue with removing the commercetools API Extension pointing to other integrations
     - removed CLI commands: `register` and `unregister`
     - added CLI commands: `api-extension-add`, `api-extension-update`, `api-extension-delete` and `api-extension-list`
-    - added new optional configuration (`COMMERCE_TOOLS_API_EXTENSION_KEY` environment variable) to recognize own commercetools API Extenion from 3rd party ones when performing delete or update operations
+    - added new optional configuration (`COMMERCE_TOOLS_API_EXTENSION_KEY` environment variable) to recognize own commercetools API Extension from 3rd party ones when performing delete or update operations
     - remove coupon from session when coupon is deleted from a cart, it requires [Sunrise Storefront v2.0.0](https://github.com/voucherifyio/sunrise-for-commerce-tools-integration)
 - 2022-07-28 `v1.0.1` Update README.md file
 - 2022-07-26 `v1.0.0` Initial release
 
 ## Migrations
 
+### Migration from v2.x.x to v3.x.x
+- replace all migrations commands `npm run migrate-products`, `npm run migrate-orders`, `npm run migrate-customers` with `npm run migrate -- --type=products`, `npm run migrate -- --type=orders`, `npm run migrate -- --type=customers`
```

Comment:
It would be great to highlight that packages need to be updated (`npm i`) as you are using the new voucherify SDK with metadata